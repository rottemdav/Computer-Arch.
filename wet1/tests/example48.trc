16 6 12 0 local_history local_tables using_share_mid
0x1cb2c N 0x5044
0xcf7ca4 N 0xf0e62b4
0x18d2c N 0x2ef85d4
0x4e69ac T 0x6628804
0xbb5bc N 0x028
0x231904 N 0x720ba0
0xcbfa0408 T 0x74c4
0x5bf458c T 0xa16db4
0xe4e0 T 0xb0f40ae4
0xca5c T 0xef0
0xe4e0 N 0xb0f40ae4
0x1cb2c N 0x5044
0xc4848 N 0x511c8
0x61469ca8 T 0x5bcdf0
0xcf7ca4 N 0xf0e62b4
0xcf7ca4 T 0xf0e62b4
0x5bf458c N 0xa16db4
0x446704 N 0x3b05c9fc
0xca5c T 0xef0
0xe0f4 N 0x5f454
0x4e69ac N 0x6628804
0x18d2c N 0x2ef85d4
0x12f20 T 0x0c032d4
0x61469ca8 N 0x5bcdf0
0xe0f4 T 0x5f454
0x5bf458c N 0xa16db4
0xc4848 T 0x511c8
0x07daf50 T 0xbd48
0xf42c N 0x2e5264
0x958c N 0x1c32c8
0xc4848 T 0x511c8
0xe0f4 N 0x5f454
0x12f20 T 0x0c032d4
0xe4e0 N 0xb0f40ae4
0xcbfa0408 T 0x74c4
0xcbfa0408 N 0x74c4
0x18d2c T 0x2ef85d4
0xf42c N 0x2e5264
0x1cb2c T 0x5044
0xcf7ca4 N 0xf0e62b4
0x4e69ac T 0x6628804
0x18d2c N 0x2ef85d4
0xcbfa0408 T 0x74c4
0xcbfa0408 T 0x74c4
0x12f20 T 0x0c032d4
0xf42c T 0x2e5264
0x79c6884 N 0xc1e84
0xcbfa0408 T 0x74c4
0x61469ca8 N 0x5bcdf0
0x958c T 0x1c32c8
0x1cb2c N 0x5044
0xe0f4 N 0x5f454
0x4e69ac T 0x6628804
0x12f20 T 0x0c032d4
0x18d2c T 0x2ef85d4
0xcf7ca4 N 0xf0e62b4
0xc4848 N 0x511c8
0xaffdc N 0x554c
0xc4848 T 0x511c8
0xcf7ca4 N 0xf0e62b4
0x79c6884 N 0xc1e84
0x231904 T 0x720ba0
0x18d2c T 0x2ef85d4
0x446704 N 0x3b05c9fc
0xca5c N 0xef0
0x07daf50 T 0xbd48
0xf42c T 0x2e5264
0xf42c N 0x2e5264
0xc4848 T 0x511c8
0x1cb2c T 0x5044
0xe4e0 N 0xb0f40ae4
0x16a0 T 0x92280
0x61469ca8 N 0x5bcdf0
0xbb5bc T 0x028
0x1cb2c N 0x5044
0x79c6884 T 0xc1e84
0x07daf50 T 0xbd48
0xc4848 T 0x511c8
0x231904 N 0x720ba0
0x16a0 N 0x92280
0x4e69ac N 0x6628804
0xaffdc T 0x554c
0xc4848 N 0x511c8
0x12f20 T 0x0c032d4
0x958c T 0x1c32c8
0xcbfa0408 N 0x74c4
0xc4848 N 0x511c8
0xcbfa0408 T 0x74c4
0x61469ca8 N 0x5bcdf0
0x79c6884 N 0xc1e84
0x07daf50 N 0xbd48
0x231904 N 0x720ba0
0x231904 N 0x720ba0
0xca5c T 0xef0
0x18d2c T 0x2ef85d4
0x07daf50 N 0xbd48
0xca5c N 0xef0
0xe0f4 T 0x5f454
0x07daf50 T 0xbd48
0xca5c T 0xef0
0xcbfa0408 N 0x74c4
0xf42c N 0x2e5264
0x446704 N 0x3b05c9fc
0x12f20 T 0x0c032d4
0x5bf458c T 0xa16db4
0x18d2c T 0x2ef85d4
0xca5c T 0xef0
0xca5c T 0xef0
0x958c T 0x1c32c8
0xe4e0 N 0xb0f40ae4
0x958c T 0x1c32c8
0xf42c T 0x2e5264
0xe4e0 N 0xb0f40ae4
0x231904 T 0x720ba0
0x231904 T 0x720ba0
0xcbfa0408 T 0x74c4
0x18d2c N 0x2ef85d4
0xca5c T 0xef0
0xe0f4 N 0x5f454
0x5bf458c T 0xa16db4
0xf42c N 0x2e5264
0x1cb2c T 0x5044
0xcf7ca4 T 0xf0e62b4
0x61469ca8 N 0x5bcdf0
0xca5c N 0xef0
0xaffdc T 0x554c
0xcbfa0408 T 0x74c4
0xcbfa0408 T 0x74c4
0x958c T 0x1c32c8
0x1cb2c T 0x5044
0xca5c N 0xef0
0x4e69ac T 0x6628804
0x12f20 T 0x0c032d4
0x16a0 N 0x92280
0xe4e0 T 0xb0f40ae4
0xe4e0 T 0xb0f40ae4
0x5bf458c T 0xa16db4
0x1cb2c N 0x5044
0x231904 N 0x720ba0
0x61469ca8 N 0x5bcdf0
0xc4848 T 0x511c8
0x1cb2c T 0x5044
0xc4848 T 0x511c8
0xbb5bc N 0x028
0xcbfa0408 T 0x74c4
0x79c6884 T 0xc1e84
0x79c6884 N 0xc1e84
0x1cb2c N 0x5044
0x07daf50 N 0xbd48
0x12f20 T 0x0c032d4
0x07daf50 N 0xbd48