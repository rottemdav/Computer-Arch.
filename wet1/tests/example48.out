0x1cb2c N 0x1cb30
0xcf7ca4 N 0xcf7ca8
0x18d2c N 0x18d30
0x4e69ac N 0x4e69b0
0xbb5bc N 0xbb5c0
0x231904 N 0x231908
0xcbfa0408 N 0xcbfa040c
0x5bf458c N 0x5bf4590
0xe4e0 N 0xe4e4
0xca5c N 0xca60
0xe4e0 N 0xe4e4
0x1cb2c N 0x1cb30
0xc4848 N 0xc484c
0x61469ca8 N 0x61469cac
0xcf7ca4 N 0xcf7ca8
0xcf7ca4 N 0xcf7ca8
0x5bf458c N 0x5bf4590
0x446704 N 0x446708
0xca5c N 0xca60
0xe0f4 N 0xe0f8
0x4e69ac N 0x4e69b0
0x18d2c N 0x18d30
0x12f20 N 0x12f24
0x61469ca8 N 0x61469cac
0xe0f4 N 0xe0f8
0x5bf458c N 0x5bf4590
0xc4848 N 0xc484c
0x7daf50 N 0x7daf54
0xf42c N 0xf430
0x958c N 0x9590
0xc4848 N 0xc484c
0xe0f4 N 0xe0f8
0x12f20 N 0x12f24
0xe4e0 N 0xe4e4
0xcbfa0408 N 0xcbfa040c
0xcbfa0408 N 0xcbfa040c
0x18d2c N 0x18d30
0xf42c N 0xf430
0x1cb2c N 0x1cb30
0xcf7ca4 N 0xcf7ca8
0x4e69ac N 0x4e69b0
0x18d2c N 0x18d30
0xcbfa0408 N 0xcbfa040c
0xcbfa0408 N 0xcbfa040c
0x12f20 N 0x12f24
0xf42c N 0xf430
0x79c6884 N 0x79c6888
0xcbfa0408 N 0xcbfa040c
0x61469ca8 N 0x61469cac
0x958c N 0x9590
0x1cb2c N 0x1cb30
0xe0f4 N 0xe0f8
0x4e69ac N 0x4e69b0
0x12f20 N 0x12f24
0x18d2c N 0x18d30
0xcf7ca4 N 0xcf7ca8
0xc4848 N 0xc484c
0xaffdc N 0xaffe0
0xc4848 N 0xc484c
0xcf7ca4 N 0xcf7ca8
0x79c6884 N 0x79c6888
0x231904 N 0x231908
0x18d2c N 0x18d30
0x446704 N 0x446708
0xca5c N 0xca60
0x7daf50 N 0x7daf54
0xf42c N 0xf430
0xf42c N 0xf430
0xc4848 N 0xc484c
0x1cb2c N 0x1cb30
0xe4e0 N 0xe4e4
0x16a0 N 0x16a4
0x61469ca8 N 0x61469cac
0xbb5bc N 0xbb5c0
0x1cb2c N 0x1cb30
0x79c6884 N 0x79c6888
0x7daf50 N 0x7daf54
0xc4848 N 0xc484c
0x231904 N 0x231908
0x16a0 N 0x16a4
0x4e69ac N 0x4e69b0
0xaffdc N 0xaffe0
0xc4848 N 0xc484c
0x12f20 N 0x12f24
0x958c N 0x9590
0xcbfa0408 N 0xcbfa040c
0xc4848 N 0xc484c
0xcbfa0408 N 0xcbfa040c
0x61469ca8 N 0x61469cac
0x79c6884 N 0x79c6888
0x7daf50 N 0x7daf54
0x231904 N 0x231908
0x231904 N 0x231908
0xca5c N 0xca60
0x18d2c N 0x18d30
0x7daf50 N 0x7daf54
0xca5c N 0xca60
0xe0f4 N 0xe0f8
0x7daf50 N 0x7daf54
0xca5c N 0xca60
0xcbfa0408 N 0xcbfa040c
0xf42c N 0xf430
0x446704 N 0x446708
0x12f20 N 0x12f24
0x5bf458c N 0x5bf4590
0x18d2c N 0x18d30
0xca5c N 0xca60
0xca5c N 0xca60
0x958c N 0x9590
0xe4e0 N 0xe4e4
0x958c N 0x9590
0xf42c N 0xf430
0xe4e0 N 0xe4e4
0x231904 N 0x231908
0x231904 N 0x231908
0xcbfa0408 N 0xcbfa040c
0x18d2c N 0x18d30
0xca5c N 0xca60
0xe0f4 N 0xe0f8
0x5bf458c N 0x5bf4590
0xf42c N 0xf430
0x1cb2c N 0x1cb30
0xcf7ca4 N 0xcf7ca8
0x61469ca8 N 0x61469cac
0xca5c N 0xca60
0xaffdc N 0xaffe0
0xcbfa0408 N 0xcbfa040c
0xcbfa0408 N 0xcbfa040c
0x958c N 0x9590
0x1cb2c N 0x1cb30
0xca5c N 0xca60
0x4e69ac N 0x4e69b0
0x12f20 N 0x12f24
0x16a0 N 0x16a4
0xe4e0 N 0xe4e4
0xe4e0 N 0xe4e4
0x5bf458c N 0x5bf4590
0x1cb2c N 0x1cb30
0x231904 N 0x231908
0x61469ca8 N 0x61469cac
0xc4848 N 0xc484c
0x1cb2c N 0x1cb30
0xc4848 N 0xc484c
0xbb5bc N 0xbb5c0
0xcbfa0408 N 0xcbfa040c
0x79c6884 N 0x79c6888
0x79c6884 N 0x79c6888
0x1cb2c N 0x1cb30
0x7daf50 N 0x7daf54
0x12f20 N 0x12f24
0x7daf50 N 0x7daf54
flush_num: 79, br_num: 151, size: 2832b
